connectors/src/connectors/microsoft/index.ts

```diff
@@ -102,10 +102,10 @@ export async function stopMicrosoftConnector(
   connectorId: ModelId
 ): Promise<Result<undefined, Error>> {
   console.log("stopMicrosoftConnector", connectorId);
-  throw Error("Not implemented");
+  return new Ok(undefined);
```

Comment:
so that we can delete the connector via poke
since the connector cannot yet be started, that's fine IMHO

Reply:
It's still needed with current main to delete connector from poke
Ok for you if I move forward, and when this becomes not needed I can revert this part?

Reply:
sure, no need to change that, fine